php_phongo.c

```diff
@@ -81,6 +81,19 @@ ZEND_DECLARE_MODULE_GLOBALS(mongodb)
 #endif
 #endif
 
+php_phongo_server_description_type_map_t
+php_phongo_server_description_type_map[PHONGO_SERVER_DESCRIPTION_TYPES] = {
+	{ PHONGO_SERVER_UNKNOWN, "Unknown" },
+	{ PHONGO_SERVER_STANDALONE, "Standalone" },
```

Comment:
As the map is from string to int, shouldn't the order be the other way around here? I.e., first string, and then the int constant?